parsl/monitoring/db_manager.py

```diff
@@ -141,6 +141,20 @@ class Task(Base):
             PrimaryKeyConstraint('task_id', 'run_id'),
         )
 
+    class Node(Base):
+        __tablename__ = NODE
+        run_id = Column('run_id', Text, nullable=False)
+        hostname = Column('hostname', Text, nullable=False)
+        cpu_count = Column('cpu_count', Integer, nullable=False)
+        total_memory = Column('total_memory', Integer, nullable=False)
+        active = Column('active', Boolean, nullable=False)
+        worker_count = Column('worker_count', Integer, nullable=False)
+        python_v = Column('python_v', Text, nullable=False)
+        reg_time = Column('reg_time', DateTime, nullable=False)
+        __table_args__ = (
+            PrimaryKeyConstraint('hostname', 'run_id'),
```

Comment:
I think this constraint is wrong.

A given host might report multiple times inside one run - consider I am allocated a node for some maximum permitted time, eg 24h, and then at the end of that, I request another allocation and am given the same node again (this seems unlikely when thinking of requesting one node on a large machine, but seems very likely when running a machine-sized workflow that runs for a long time)